aten/src/ATen/ParallelNative.cpp

```diff
@@ -212,9 +203,7 @@ int get_num_threads() {
     return _get_intraop_pool().size() + 1;
   }
 #else
-  caffe2::ThreadPool* pool = caffe2::mobile_threadpool();
-  // caffe2::ThreadPool::getNumThreads() counts the current thread.
-  return !pool || in_parallel_region() ? 1 /* current thread */ : pool->getNumThreads();
```

Comment:
@ilia-cher Can you explain what in_parallel_region() is for?  To avoid over parallelization (!?) when invoked from a parallel context?  This check was added to avoid a deadlock while we were using caffe2::ThreadPool if I remember correctly, but now that we are moving away from that, is this still needed? cc @ljk53